crates/iota-tool/src/lib.rs

```diff
@@ -107,24 +106,19 @@ async fn make_clients(
     net_config.connect_timeout = Some(Duration::from_secs(5));
     let mut authority_clients = BTreeMap::new();
 
-    let active_validators = match iota_client
+    let state = iota_client
         .governance_api()
         .get_latest_iota_system_state()
-        .await?
-    {
-        IotaSystemStateSummary::V1(v1) => v1.active_validators,
-        IotaSystemStateSummary::V2(v2) => v2.iter_committee_members().cloned().collect::<Vec<_>>(),
-        _ => panic!("unsupported IotaSystemStateSummary"),
-    };
+        .await?;
 
-    for validator in active_validators {
-        let net_addr = Multiaddr::try_from(validator.net_address).unwrap();
+    for committee_member in state.iter_committee_members() {
```

Comment:
comment missing why we use the committee here.

Reply:
Refreshed comment to `make_clients` function.